generated/nifgen/nifgen/_grpc.py

```diff
@@ -0,0 +1,514 @@
+# -*- coding: utf-8 -*-
+# This file was generated
+
+import grpc
+import hightime  # noqa: F401
+import threading
+import warnings
+
+from . import enums as enums
+from . import errors as errors
+from . import nifgen_pb2 as grpc_types
+from . import nifgen_pb2_grpc as nifgen_grpc
+
+
+class GrpcStubInterpreter(object):
```

Comment:
await: Why isn't the file `session.py` included in this PR? It's the class that will instantiate this.

Reply:
Something appeared to be in the wrong state on my machine. I deleted the folder `nimi-python\src\generated\nifgen`, and re-ran the codegen and this did cause `session.py` to be edited. I've just pushed the changes to this PR.

Reply:
I'm going to abandon this change and re-create the PR under a new branch, because git has gotten into a bad state on my machine.